### PR TITLE
bump version and use recommended numpy build matrix

### PR DESCRIPTION
**Pull request by marscher**

### CODE REVIEW

**Comment by mpharrigan** [comment]
LGTM

@marscher do you want to add yourself as a maintainer so you can merge this / future PRs?

**Comment by marscher** [comment]
thanks for the offer, but no thanks :)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+{% set sha256 = "a5d1922b1db6f4c3012416e7dc14de2984b9335a48c895e1698afd5c718312b0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/1e/00/9910dd324f32582051d0ee6922c3cd4727234aae96366f3867c46a70cd78/fastcluster-1.1.24.tar.gz#md5=e71235732f43f5f19d71cfaf3b45ff0c
```
Comment (isuruf) [inline recipe/meta.yaml]: Why this change?

**Comment by marscher** [comment]
> -  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
> +  url: https://pypi.python.org/packages/1e/00/9910dd324f32582051d0ee6922c3cd4727234aae96366f3867c46a70cd78/fastcluster-1.1.24.tar.gz#md5=e71235732f43f5f19d71cfaf3b45ff0c
> 
> Why this change?
> 
PyPi has abandoned the old naming scheme in favour of this one. I
currently can not find the announcement, but this is the actual state.
When they introduced it, the old scheme was not working, but now it
seems they have fixed this (added a redirect to the old scheme).

Gonna revert that change. Thanks!

**Comment by mpharrigan** [comment]
ping me when travis completes and I will merge

**Comment by marscher** [comment]
this is going to take a while, because the osx backlog of travis is
super long.

**Comment by mpharrigan** [comment]
I'd prefer to wait just to be safe. Is this urgent?

**Comment by marscher** [comment]
On 12/14/2017 06:43 PM, Matthew Harrigan wrote:
> I'd prefer to wait just to be safe. Is this urgent?
no, we can wait
